torch/csrc/onnx/onnx.h

```diff
@@ -20,5 +20,5 @@ enum class TrainingMode {
 // onnx::IR_VERSION. with this change, the test_operators.py will be more
 // stable. only bump it when it's necessary
 static const size_t IR_VERSION = 6;
-static const char* PRODUCER_VERSION = "1.7";
+static const char* PRODUCER_VERSION = "1.8";
```

Comment:
does this change not affect expect file tests?

Reply:
No, test files have a placeholder for this variable.

Reply:
https://github.com/pytorch/pytorch/blob/1fd48a9d1f4158fb015793caa282d2c0ca92059d/test/onnx/expect/TestOperators.test_acos.expect#L3